test/ERC20/common-ERC20-issues.ts

```diff
@@ -641,24 +643,6 @@ describe('common-erc20-issues', () => {
       want((await gem.allowance(ALI, ALI)).toNumber()).to.equal(0);
     })
 
-    // Non string metadata (TH)
```

Comment:
Deleted because we're intentionally doing what this test was intended to prevent.